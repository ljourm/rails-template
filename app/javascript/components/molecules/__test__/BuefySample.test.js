import { createLocalVue, shallowMount } from "@vue/test-utils"
import Buefy from "buefy"
import Component from "../BuefySample"

describe("Component", () => {
  const localVue = createLocalVue()
  localVue.use(Buefy)

  const wrapper = shallowMount(Component, {
    localVue,
    mocks: {
      $store: {
        state: {
          loading: false,
        },
      },
    },
  })

  test("snapshot", () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
