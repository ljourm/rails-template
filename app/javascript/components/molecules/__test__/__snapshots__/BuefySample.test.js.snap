// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component snapshot 1`] = `
<div
  class="buefy-sample"
>
  <p>
    Buefy Sample
  </p>
  <b-field-stub
    addons="true"
    label="Label"
  >
    <b-input-stub
      customclass=""
      hascounter="true"
      type="text"
      usehtml5validation="true"
      value="input value"
    />
  </b-field-stub>
  <b-button-stub
    class="is-primary"
    nativetype="button"
    tag="button"
  >
    button
  </b-button-stub>
  <span
    class="icon has-text-info"
  >
    <i
      class="far fa-address-book"
    />
    <i
      class="fas fa-info-circle"
    />
  </span>
</div>
`;
